MAUI/NavigationDrawer/Side-Pane-Content.md

```diff
@@ -0,0 +1,363 @@
+---
+layout: post
+title: Setting Sliding Panel Content in .NET MAUI NavigationDrawer | Syncfusion
+description: Learn here all about Setting Sliding Panel Content support in Syncfusion .NET MAUI Navigation Drawer (SfNavigationDrawer) control and more.
+platform: maui
+control: NavigationDrawer
+documentation: ug
+---
+
+
+# Setting Sliding Panel Content in .NET MAUI Navigation Drawer
+
+The drawer pane content is viewable only if the drawer is in the open state. Its content can be set as
+
+*	Header Content
+
+*	Drawer Content
+
+*	Footer Content
+
+N> Header and Footer content are optional but Drawer content is mandatory to allocate space for the drawer.
+		
+## Header Content
+
+As the name suggests it is displayed at the top of drawer. [`DrawerHeaderView`]() property is used to set the header content of drawer. 
```

Comment:
As the name suggests it is displayed at the top of drawer. The `DrawerHeaderView` property is used to set the header content of the drawer.

Reply:
Changed